packages/api/src/external/fhir/shared/index.ts

```diff
@@ -25,14 +25,14 @@ export function downloadedFromHIEs(doc: DocumentReference): boolean {
 
 // Creates a FHIR data query string based on the specified range.
 // For example, if dateFrom="2022-03-23" & dateTo="2024-01-02", result will look like:
-//  "date=sa2022-03-23&date=le2024-01-02"
+//  "date=ge2022-03-23&date=le2024-01-02"
 //
 // See details here: https://www.hl7.org/fhir/R4/search.html#date
 export function isoDateRangeToFHIRDateQuery(dateFrom?: string, dateTo?: string): string {
   const fhirDateQueryBase = "date=";
   let fhirDateQuery = `${fhirDateQueryBase}`;
   if (!dateFrom && !dateTo) return "";
-  if (dateFrom) fhirDateQuery += `${Operator.STARTS_AFTER}${dateFrom}`;
+  if (dateFrom) fhirDateQuery += `${Operator.GREATER_THAN_OR_EQUALS}${dateFrom}`;
```

Comment:
Changed from `sa` (starts after) to `ge` (greater than or equals) because our docs state that parameter is inclusive on the value provided: https://docs.metriport.com/medical-api/api-reference/fhir/get-patient-consolidated

Reply:
Let's not change this, `ge` will not have the desired effect when filtering - I forgot exactly why, but I used `ge` initially and it didn't work as expected in certain cases due to FHIR specifics.

Reply:
At best, I'd suggest we raise some other issue to look into this later

Reply:
Ok, I reverted it. It was originally changed from `ge` to `sa` on [this commit](https://github.com/metriport/metriport/commit/c2958f4736a0335174b6ebab4fcb9166b415da69).

It would be good to document these odd cases so we don't have to reinvestigate it again. What I can get from the PR message (and code) is that we're including results with date after the start date (exclusive), but on the docs we say its inclusive.

I'll create a ticket to review this.